api/services/get.js

```diff
@@ -5,18 +5,18 @@ module.exports = self;
 
 var async = require('async');
 var _ = require('underscore');
-var fs = require('fs');
-var path = require('path');
-
 var configHandler = require('../../common/configHandler.js');
+var path = require('path');
 
 function get(req, res) {
   var bag = {
     reqQuery: req.query,
     resBody: [],
     initializeDefault: false,
     component: 'services',
-    defaultService: {}
+    defaultService: {},
+    services: require(path.join(global.config.scriptsDir,
```

Comment:
@ric03uec : `Require` parses the JSON files by default, I don't think it's adding any value by adding error handling for JSON files.
If we are doing that we are giving developers the option to break the `services.json`.